data_safe_haven/external/api/azure_api.py

```diff
@@ -28,38 +29,43 @@
 from azure.mgmt.dns import DnsManagementClient
 from azure.mgmt.dns.models import RecordSet, TxtRecord
 from azure.mgmt.keyvault import KeyVaultManagementClient
-from azure.mgmt.keyvault.models import AccessPolicyEntry, Permissions
-from azure.mgmt.keyvault.models import Sku as KeyVaultSku
 from azure.mgmt.keyvault.models import (
+    AccessPolicyEntry,
+    Permissions,
     Vault,
     VaultCreateOrUpdateParameters,
     VaultProperties,
 )
+from azure.mgmt.keyvault.models import Sku as KeyVaultSku
 from azure.mgmt.msi import ManagedServiceIdentityClient
 from azure.mgmt.msi.models import Identity
 from azure.mgmt.resource import ResourceManagementClient, SubscriptionClient
 from azure.mgmt.resource.resources.models import ResourceGroup
 from azure.mgmt.storage import StorageManagementClient
 from azure.mgmt.storage.models import (
     BlobContainer,
-    Kind as StorageAccountKind,
     PublicAccess,
-    Sku as StorageAccountSku,
     StorageAccount,
     StorageAccountCreateParameters,
     StorageAccountKey,
     StorageAccountListKeysResult,
 )
+from azure.mgmt.storage.models import (
+    Kind as StorageAccountKind,
+)
+from azure.mgmt.storage.models import (
+    Sku as StorageAccountSku,
+)
```

Comment:
These could/should be in one include block. What's the reason for separating them?